packages/beacon-node/src/chain/produceBlock/produceBlockBody.ts

```diff
@@ -281,6 +291,7 @@ export async function prepareExecutionPayload(
 }
 
 async function prepareExecutionPayloadHeader(
+  seq: ForkSeq,
```

Comment:
name variable `fork`, seq is way to generic. If there's another fork variable, then forkSeq